loops_lab.js

```diff
@@ -0,0 +1,208 @@
+// # Loops Lab Exercises
```

Comment:
I can see that you started to understand loops fairly well. Good practice as a developer is that the less loops we use, the better. I see instances like last question where multiple loops were used to solve the problem. Try to limit the number of loops used to 2 at max. Keep up the good work!